src/PrestaShopBundle/Command/CheckTranslationDuplicatesCommand.php

```diff
@@ -27,13 +27,25 @@
 namespace PrestaShopBundle\Command;
 
 use PrestaShopBundle\Translation\Translator;
-use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Helper\ProgressBar;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Translation\TranslatorBagInterface;
 
-class CheckTranslationDuplicatesCommand extends ContainerAwareCommand
+class CheckTranslationDuplicatesCommand extends Command
 {
+    /**
+     * @var TranslatorBagInterface
+     */
+    private $translator;
+
+    public function __construct(TranslatorBagInterface $translator)
```

Comment:
Please document BC breaks in constructor

Reply:
I already documented every command constructor now takes parameters, should I document them individually?

Reply:
If you have a little time, yes, it would be better.